andela/Ah-backend-aquaman
# #164069235 User should be able to see the time it takes to read an article

## What does this PR do?
This PR adds a feature that enables a user to see the time it takes to read an article.

## How should this be manually tested?

- Clone this repository
- cd into the directory Ah-backend-aquaman
- Checkout the branch ft-article-read-time-164069235
- Create and activate your virtual environment
- pip install -r requirements.txt
- python manage.py makemmigrations
- python manage.py migrate
- Source your .env file
- python manage.py runserver
- Sign up, verify your email and login
- Then navigate to the endpoint http://127.0.0.1:8000/api/articles/ and post an article
- Thereafter using the same endpoint http://127.0.0.1:8000/api/articles/ get back the articles
- You will be able to see the readtime associated with that article

## What are the relevant pivotal tracker stories?

[#164069235](https://www.pivotaltracker.com/story/show/164069235)

## Screenshots

After creating your account, verifying your account and logging in, copy the token and paste it into the token field

![image](https://user-images.githubusercontent.com/42435346/54342175-d9f9db00-464c-11e9-85c0-6526a338ecc6.png)

Then proceed to the articles endpoint and post an article

![image](https://user-images.githubusercontent.com/42435346/54342487-a10e3600-464d-11e9-84bb-5fbdecdaf06f.png)

When you get back the articles, you are able to see the time it takes to read that article

![image](https://user-images.githubusercontent.com/42435346/54342366-4ecd1500-464d-11e9-9efd-dd74ab4201db.png)
